LLVMInsTrim.cpp

```diff
@@ -31,6 +31,9 @@ namespace {
       return generator() % 65536;
     }
 
+ protected:
+  uint32_t function_minimum_size = 1;
```

Comment:
nit: The indent is not aligned for these two lines.
nit: function_minimum_size -> FunctionMinimumSize (we try to follow the LLVM coding style.)